test/select.jl

```diff
@@ -2707,4 +2707,57 @@ end
     @test_throws ArgumentError select(gdf, :a => [:x, :y, :z])
 end
 
+@testset "additional test of handling of operation specfication in select!/transform!" begin
+    df = DataFrame(a=1:4, b='a':'d', c=["p", "q", "r", "s"])
+    select!(df, :b, :c => :d, :a => (x -> 2 * x) => :e)
+    @test df == DataFrame(b='a':'d', d=["p", "q", "r", "s"], e=[2, 4, 6, 8])
+
+    df = DataFrame(a=1:4, b='a':'d', c=["p", "q", "r", "s"])
+    transform!(df, :b, :c => :d, :a => (x -> 2 * x) => :e)
+    @test df == DataFrame(a=1:4, b='a':'d', c=["p", "q", "r", "s"],
+                          d=["p", "q", "r", "s"], e=[2, 4, 6, 8])
+    @test df.c !== df.d
+
+    df = DataFrame(a=1:4, b='a':'d', c=["p", "q", "r", "s"])
+    select!(groupby(df, :c), :b, :c => :d, :a => (x -> 2 * x) => :e)
+    @test df == DataFrame(c=["p", "q", "r", "s"], b='a':'d',
+                          d=["p", "q", "r", "s"], e=[2, 4, 6, 8])
+    @test df.c !== df.d
```

Comment:
No need to check aliasing in other places?

Reply:
In the new tests I test for aliasing for `GroupedDataFrame` everywhere it makes sense (is there any point I missed?).

Reply:
No, I just wanted to check.